content/getting-started/signup.md

```diff
@@ -26,10 +36,10 @@ Read more about [GitHub organizations and OAuth app restrictions](https://help.g
 
 ## Sign up via Codemagic GitHub app
 
-The GitHub app integration has several advantages over the OAuth integration as you can grant Codemagic access to select repositories and only **read** access to your code is required.
+The GitHub app integration has several advantages over the OAuth integration as you can grant Codemagic access to selected repositories and only **read** access to your code is required.
 
-1. Click **Join using GitHub App** at [codemagic.io/signup](https://codemagic.io/signup). You will be authorized via GitHub without granting any access to your repositories. 
-2. In Codemagic, go to **User settings > Integrations** and click **Install GitHub App**. You will be redirected to GitHub to select the repositories you want to grant access to and finish the installation. 
+1. Click **GitHub** at [codemagic.io/signup](https://codemagic.io/signup) and select **Selected repositories only** to sign up via Codemagic GitHub app. Click **Authorize Codemagic** to verify your identity. You will be authenticated via GitHub without granting any access to your repositories. 
+2. In Codemagic, go to **User settings > Integrations** and click **Connect**. You will be redirected to GitHub to select the repositories you want to grant access to and finish the installation. 
```

Comment:
Is the button name **Connect** in all cases or is it **Install GitHub App** when it's not installed yet?